site/en/docs/extensions/mv3/tut_websockets/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Using WebSockets in Service Workers'
+seoTitle: 'How to use WebSockets in Chrome extension Service Workers'
+date: 2023-07-04
+description: 'Step-by-step instructions on how to connect to a WebSocket in your Chrome extension.'
+---
+
+This tutorial demonstrates how to connect to a WebSocket in your Chrome extension's service worker. You can find a [working example on Github](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/tutorial.websockets).
+
+## Background {: #background }
+
+Starting with Chrome 116, extension service workers get improved support for [WebSockets[(https://developer.mozilla.org/docs/Web/API/WebSockets_API). Previously, a service worker could become inactive despite a WebSocket connection being active if no other extension events occurred for 30 seconds. This would terminate the service worker and close the WebSocket connection. For more background on the extension service worker lifecycle, read the [extension service worker guide](/docs/extensions/mv3/service_workers/service-worker-lifecycle/#idle-shutdown)).
```

Comment:
```suggestion
Starting with Chrome 116, extension service workers get improved support for [WebSockets](https://developer.mozilla.org/docs/Web/API/WebSockets_API). Previously, a service worker could become inactive despite a WebSocket connection being active if no other extension events occurred for 30 seconds. This would terminate the service worker and close the WebSocket connection. For more background on the extension service worker lifecycle, read the [extension service worker guide](/docs/extensions/mv3/service_workers/service-worker-lifecycle/#idle-shutdown)).
```
Link fix